package.json

```diff
@@ -12,11 +13,17 @@
     "url": "http://tareksherif.net/"
   },
   "license": "MIT",
+  "scripts": {
+    "build": "browserify --standalone PicoGL -o build/picogl.js src/index.js && npm run compress",
+    "compress": "uglifyjs build/picogl.js --compress --mangle > build/picogl.min.js"
+  },
```

Comment:
I've stopped using grunt,gulp, etc. so I'm a little rusty on the syntax. Would you mind making this change?